data-sources/gigamon-appinsights/config.yml

```diff
@@ -0,0 +1,17 @@
+id: gigamon-appinsights
+displayName: Gigamon Application Insights
+description: |
+  Gigamon Application Intelligence gives NetOps, CloudOps, and security teams full visibility into applications running in datacenters and public clouds. Therefore, you can maintain consistently strong security across your environment plus get to the root causes of vulnerabilities and performance bottlenecks.
```

Comment:
Looks like this description is a bit long, we need to cut it down to under 240 characters.

Reply:
Cut down the description